### PR TITLE
en,zh: add release notes for v1.2.3

### PR DESCRIPTION
**Pull request by KanShiori**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/releases/release-1.2.3.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+# TiDB Operator 1.2.3 Release Notes
+
+发布日期：2021 年 9 月 7 日
+
+TiDB Operator 版本：1.2.3
+
+## Bug fixes
+
+- 修复升级 TiDBOperator v1.2.2 导致 TiFlash Pod 重建的问题 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```
Comment (DanielZhangQD) [inline zh/releases/release-1.2.3.md:13]:
```suggestion
- 修复升级 TiDB Operator v1.2.2 导致 TiFlash Pod 滚动重启的问题 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.2.3.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# TiDB Operator 1.2.3 Release Notes
+
+Release date: September 7, 2021
+
+TiDB Operator version: 1.2.3
+
+## Bug fixes
+
+- Fix the recreation of the TiFlash Pod after upgrade TiDBOperator to v1.2.2 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```
Comment (qiancai) [inline en/releases/release-1.2.3.md:13]:
```suggestion
- Fix the issue that the TiFlash Pod is recreated after TiDBOperator is upgraded to v1.2.2 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```

File: zh/releases/release-1.2.3.md (hunk 1 above)
Comment (qiancai) [inline zh/releases/release-1.2.3.md:13]:
```suggestion
- 修复升级到 TiDBOperator v1.2.2 时导致 TiFlash Pod 重建的问题 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```

File: en/releases/release-1.2.3.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+# TiDB Operator 1.2.3 Release Notes
+
+Release date: September 7, 2021
+
+TiDB Operator version: 1.2.3
+
+## Bug fixes
+
+- Fix the issue that the TiFlash Pod rolling recreate after TiDB Operator is upgraded to v1.2.2 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```
Comment (qiancai) [inline en/releases/release-1.2.3.md:13]:
```suggestion
- Fix the TiFlash Pod rolling recreation issue after TiDB Operator is upgraded to v1.2.2 ([#4181](https://github.com/pingcap/tidb-operator/pull/4181), [@KanShiori](https://github.com/KanShiori))
```

**Review by qiancai — APPROVED** [review]
LGTM

**Comment by qiancai** [comment]
I've finished the self-check of this PR. PTAL

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Pull request merged**